src/filters/appointmentsFilter.js

```diff
@@ -22,10 +22,9 @@ angular.module('bahmni.appointments')
                 if ((appointment.providers == undefined) || _.isEmpty(appointment.providers)) {
                     return _.includes(providerUuids, 'no-provider-uuid');
                 } else {
-                    var found = _.find(appointment.providers, function (p) {
-                        return _.includes(providerUuids, p.uuid);
+                    return _.find(appointment.providers, function (p) {
```

Comment:
I dont understand the difference, really.
Also this is supposed to return a boolean value. _.find returns the first matching element, not boolean.